frontend/src/model/Meeple.ts

```diff
@@ -0,0 +1,17 @@
+import Locations from '../constants/locations';
+class Player {}
+class Tile {}
+
+class Meeple {
+  playerRef: Player;
+  isPlaced: boolean;
```

Comment:
I wonder if we could change this property to be a getter instead - now we need to change both tile and isPlaced on Meeple placement